src/index.js

```diff
@@ -62,6 +65,68 @@ const createContext = (ctx) => {
   return ctx
 }
 
+const addTypeScriptLoader = (options = {}, loader) => {
+  const moduleName = 'postcss'
+
+  return {
+    ...options,
+    searchPlaces: [
+      ...(options.searchPlaces || []),
+      'package.json',
+      `.${moduleName}rc`,
+      `.${moduleName}rc.json`,
+      `.${moduleName}rc.yaml`,
+      `.${moduleName}rc.yml`,
+      `.${moduleName}rc.ts`,
+      `.${moduleName}rc.js`,
+      `${moduleName}.config.ts`,
+      `${moduleName}.config.js`
```

Comment:
```suggestion
      `${moduleName}.config.cjs`
```

Reply:
Do you want me to add this, or replace it? the latter will remove all support for `.config.js`, which doesn't seem right to do in this PR.

Reply:
Oops. I fixed the suggestion (let's add `.cjs` too)